V2_METRICS.md

```diff
@@ -0,0 +1,74 @@
+# Get Metric
+
+Returns data for a given metric. The input, that is needed, is the string 'metric/slug', a 'from' date, a 'to' date, a string for the interval and an aggregation, which is optional (It is 'NULL' by default).
```

Comment:
I think only the available metrics should stay in this file. If you want to explain the options it should be in the README.md before the example. Also `aggregation` - when it it not passed as argument it uses the aggregation determined from the API.

Reply:
You're right, done as you've said